packages/node/src/generators/application/application.spec.ts

```diff
@@ -459,4 +459,24 @@ describe('app', () => {
       expect(devkit.formatFiles).not.toHaveBeenCalled();
     });
   });
+
+  describe.each([
+    ['fastify' as const, true],
+    ['express' as const, false],
+    ['koa' as const, false],
```

Comment:
Do you need the last two entries?
```
    ['express' as const, false],
    ['koa' as const, false],
```

Reply:
Yeah, because we should at least check that the target is added. I thought about adding unit tests too, but it's not possible without another library I think.

Reply:
okay i see.